### PR TITLE
CN 3772 cel tier stats

### PR DESCRIPTION
**Pull request by srkela**
PR Checklist:
- [x] I have named the PR correctly
- [x] I have added at least two reviewers
- [x] I have assigned the PR to myself
- [x] I have merged develop into my feature branch and resolved conflicts
- [x] I have tested all new features locally
- [x] I have tested relevant flow locally

### How to name a PR
- If it includes only one issue [CN-1234 Issue name]
- If it includes many issues [CN-1234, CN-2345, CN-3456 What was updated/changed/added]
- If the PR is not ready yet, add [DRAFT] in front of the name

### How to test locally
1. Test happy path
2. Test other paths
3. Remove all warnings

### How to relevant flow locally
- If working on part of CelPay, check the whole happy path

### CODE REVIEW

File: app/components/organisms/CelStats/CelStats.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+// import * as appActions from "../../../redux/actions";
+import CelsiusStatsStyle from "./CelStats.styles";
+import CelText from '../../atoms/CelText/CelText';
+import Separator from "../../atoms/Separator/Separator";
+import STYLES from "../../../constants/STYLES";
+import { THEMES } from '../../../constants/UI'
+import formatter from "../../../utils/formatter";
+import { getTheme } from '../../../utils/styles-util'
+
+// @connect(
```
Comment (fjovakaric) [inline app/components/organisms/CelStats/CelStats.js]: remove redux

File: app/components/organisms/CelStats/CelStats.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+//   state => ({
+//
+//   }),
+//   dispatch => ({ actions: bindActionCreators(appActions, dispatch) }),
+// )
+class CelStats extends Component {
+
+  static propTypes = {
+    // text: PropTypes.string
```
Comment (fjovakaric) [inline app/components/organisms/CelStats/CelStats.js]: add `celTierStats`

File: app/components/organisms/CelStats/CelStats.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                type={'H7'}
+                align={'center'}
+            >
+              Receiving Interest in CEL
+            </CelText>
+          </View>
+        </View>
+
+        { tierStats.map((i, k) => (
+            <View style={ style.celTierWrapper} key={k}>
```
Comment (fjovakaric) [inline app/components/organisms/CelStats/CelStats.js]: move to render method

File: app/components/organisms/CelStats/CelStats.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+
+  render() {
+    const style = CelsiusStatsStyle();
+    const theme = getTheme()
+
+    const TEXT_LENGTH = 80
+    const TEXT_HEIGHT = 22
+    const OFFSET = TEXT_LENGTH / 2 - TEXT_HEIGHT / 2
+
```
Comment (fjovakaric) [inline app/components/organisms/CelStats/CelStats.js]: `return null` if no cel tier stats are passed

File: app/components/organisms/CelStats/CelStats.js [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+
+
+    // MOCK DATA
+    const tierStats = [
```
Comment (fjovakaric) [inline app/components/organisms/CelStats/CelStats.js]: remove when finished

File: app/components/organisms/CelStats/CelStats.js (hunk 4 above)
Comment (srkela) [reply app/components/organisms/CelStats/CelStats.js]: done

**Pull request merged**